package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "build": "gatsby build",
     "clean": "gatsby clean",
-    "start": "gatsby develop -o"
+    "start": "gatsby develop -o --host=0.0.0.0"
```

Comment:
curious what the advantage is?

Reply:
so veiko could see it in our local network ;)